stockflux-securities-master/src/enums.js

```diff
@@ -1,13 +1,15 @@
 export const TableState = {
   LOADING: "loading",
-  DELETING: "deleting",
+  UPDATING: "updating",
```

Comment:
I thought these enums had changed to be functional classes to allow you to use the `instanceOf` prop type and operator? Right now, the `instanceOf` prop type won't work.

Reply:
The functional classes were only for the button sizes ("extra-small", "small", "large") and for alert type ("error", "success"). The instanceOf propType still works in these components and these enums are being used in the same way as before (they are not used as props, just as enums for the state)